test/test-file-creation.js

```diff
@@ -476,49 +478,6 @@ describe('angular-fullstack generator', function () {
       }
     });
 
-    describe('with Babel ES6 preprocessor', function() {
-      beforeEach(function() {
-        helpers.mockPrompt(gen, {
-          script: 'js',
-          babel: true,
-          markup: 'jade',
-          stylesheet: 'less',
-          router: 'uirouter'
-        });
-      });
-
-      it('should run client tests successfully', function(done) {
-        this.timeout(60000);
-        gen.run({}, function () {
-          exec('grunt test:client', function (error, stdout, stderr) {
-            expect(stdout, 'Client tests failed \n' + stdout ).to.contain('Executed 1 of 1 SUCCESS');
-            done();
-          });
-        });
-      });
-
-      it('should pass jshint', function(done) {
-        this.timeout(60000);
-        gen.run({}, function () {
-          exec('grunt jshint', function (error, stdout, stderr) {
-            expect(stdout).to.contain('Done, without errors.');
-            done();
-          });
-        });
-      });
-
-      it('should run server tests successfully', function(done) {
-        this.timeout(60000);
-        gen.run({}, function () {
-          exec('grunt test:server', function (error, stdout, stderr) {
-            expect(stdout, 'Server tests failed (do you have mongoDB running?) \n' + stdout).to.contain('Done, without errors.');
-            done();
-          });
-        });
-      });
-    });
-
-
```

Comment:
I added the `babel` option to the first round of tests since its actually a default and that is what the first round is testing. Also this block was failing because it was missing choices from the ` helpers.mockPrompt` options.